azure-pipelines.yml

```diff
@@ -254,3 +254,27 @@ jobs:
       continueOnError: true
       condition: not(succeeded())
 
+- job: RichCodeNav_Indexing
```

Comment:
Is there a way to limit this job to "master" builds only (no PRs, no other branches)

Reply:
I have changed this job to only run for master CI builds, but it won't run on this PR anymore so here is its succeeded [previous build](https://dev.azure.com/dnceng/public/_build/results?buildId=647803&view=logs&jobId=c9b5975f-9534-57a6-63b4-9ff88d3ad797&j=c9b5975f-9534-57a6-63b4-9ff88d3ad797&t=9bd201d1-e2ba-5767-1cb9-95346b719bcf).